services/chaindb/postgresql/upgrader.go

```diff
@@ -1496,12 +1511,6 @@ CREATE INDEX IF NOT EXISTS i_block_withdrawals_3 ON t_block_withdrawals(f_valida
 		return errors.Wrap(err, "failed to create block withdrawals index 3")
 	}
 
-	if _, err := tx.Exec(ctx, `
-CREATE INDEX IF NOT EXISTS i_block_withdrawals_4 ON t_block_withdrawals(f_address);
-`); err != nil {
-		return errors.Wrap(err, "failed to create block withdrawals index 4")
-	}
-
```

Comment:
Why is this index being removed?

Reply:
Not sure.. probably a mistake. Readded it.